raiden/tests/integration/long_running/test_token_networks.py

```diff
@@ -258,3 +258,54 @@ def test_participant_selection(raiden_network, token_addresses):
             channel_ids=channel_identifiers,
             retry_timeout=0.1,
         )
+
+
+@raise_on_failure
+@pytest.mark.parametrize("number_of_nodes", [2])
+@pytest.mark.parametrize("channels_per_node", [0])
+@pytest.mark.parametrize("settle_timeout", [10])
+@pytest.mark.parametrize("reveal_timeout", [3])
+def test_connect_does_not_open_channels_with_offline_nodes(raiden_network, token_addresses):
+    """
+    Test that using the connection manager to connect to a token network
+    does not open channels with offline nodes
+
+    Test for https://github.com/raiden-network/raiden/issues/5583
+    """
+    # pylint: disable=too-many-locals
+    registry_address = raiden_network[0].raiden.default_registry.address
+    token_address = token_addresses[0]
+    app0 = raiden_network[0]
```

Comment:
Why not `app0, app1 = raiden_network`?

Reply:
Hmm it's just a remnant of a more complicated version of the test. I can adjust it.